terraform/workspace-variables/pen.tfvars

```diff
@@ -0,0 +1,30 @@
+# PaaS
+paas_app_environment       = "pen"
+env_config                 = "pen"
+paas_space_name            = "bat-staging"
+paas_postgres_service_plan = "tiny-unencrypted-11"
```

Comment:
restore from snapshot option won't work for tiny plan..
snapshots are only available for small plans and above.

Reply:
Thanks, as discussed with @misaka , we will build an empty database instead